megalinter/descriptors/swift.megalinter-descriptor.yml

```diff
@@ -19,16 +19,16 @@ linters:
     cli_lint_extra_args:
       - "lint"
     cli_lint_extra_args_after:
-      - "--path"
-      - "/tmp/lint"
       - "--strict"
     cli_lint_fix_arg_name: "--fix"
     cli_lint_fix_remove_args:
       - "lint"
     cli_docker_image: norionomura/swiftlint
```

Comment:
Could we use realm swiftlint's Docker [image](https://github.com/realm/SwiftLint/tree/main?tab=readme-ov-file#docker) instead?